frontend/src/teamLogoUrl.js

```diff
@@ -12,12 +13,16 @@ const knownLogos = new Map([
     ['itandroids', require('@/assets/logos/itandroids.png')],
     ['kiks', require('@/assets/logos/kiks.png')],
     ['luhbots', require('@/assets/logos/luhbots.png')],
+    ['maracatronics', require('@/assets/logos/maracatronics.png')],
     ['mct-susano-logics', require('@/assets/logos/mct-susano-logics.png')],
     ['mrl', require('@/assets/logos/mrl.png')],
     ['namec', require('@/assets/logos/namec.png')],
+    ['neon', require('@/assets/logos/neon.png')],
     ['neuislanders', require('@/assets/logos/neuislanders.png')],
     ['op-amp', require('@/assets/logos/op-amp.png')],
     ['orcabot', require('@/assets/logos/orca-bot.png')],
+    ['pequi-mecanico', require('@/assets/logos/pequi-mecanico.png')],
```

Comment:
This does not match the name in GC: `Mecânico`. Like with `robôcin`, you should add the special character to the key.